cob_phidgets/ros/src/phidgets_range_sensors.cpp

```diff
@@ -21,7 +21,9 @@ class Sensor
 	Sensor(const std::string &fr_id, const int id, const int filter_size = 10) :
 			id_(id), filter_size_(filter_size), frame_id_(fr_id)
 	{
-		pub_range_ = n_.advertise<sensor_msgs::Range>(frame_id_, 0);
+		char buffer[256];
+		sprintf(buffer, "range_%d", id);
+		pub_range_ = n_.advertise<sensor_msgs::Range>(buffer, 0);
```

Comment:
Does this change the name of the topic for cob's or raw's?
Isn't ```frame_id``` configurable via parameters?
@ipa-lth wouldn't the parameter be an option for you?